apps/meteor/client/hooks/useEndpointAction.ts

```diff
@@ -24,7 +24,7 @@ export const useEndpointAction = <TMethod extends Method, TPath extends PathFor<
 
 			return data;
 		} catch (error) {
-			dispatchToastMessage({ type: 'error', message: String(error) });
+			dispatchToastMessage({ type: 'error', message: String(error.error) });
```

Comment:
idk why we need to use `String` here :eyes:

Reply:
Should be handled here https://github.com/RocketChat/Rocket.Chat/blob/0c7034f1bb0e77eb058fcf459c8b566baf94debc/apps/meteor/client/lib/utils/handleError.ts#L27

Reply:
@dudanogueira ping to check this 😬 would be good to ship with 5.x :)

Reply:
```suggestion
			dispatchToastMessage({ type: 'error', message: error });
```